src/Components/Logo/InfluxDataLogo.tsx

```diff
@@ -0,0 +1,119 @@
+// Libraries
+import React, {forwardRef} from 'react'
+import classnames from 'classnames'
+
+// Types
+import {StandardFunctionProps, InfluxColors} from '../../Types'
+
+// Styles
+import './InfluxDataLogo.scss'
+
+export interface InfluxDataLogoProps extends StandardFunctionProps {
+  /** Coloration of the SVG image */
+  fill?: InfluxColors | string
+}
+
+export type InfluxDataLogoRef = SVGSVGElement
+
+export const InfluxDataLogo = forwardRef<
+  InfluxDataLogoRef,
+  InfluxDataLogoProps
+>(
+  (
+    {
+      id,
+      style,
+      fill = InfluxColors.White,
+      testID = 'logo--influxdata',
+      className,
+    },
+    ref
+  ) => {
+    const logoClass = classnames('cf-logo--influxdata', {
+      [`${className}`]: className,
+    })
+
+    const logoStyle = {fill, ...style}
+
+    return (
+      <svg
+        version="1.1"
+        ref={ref}
+        id={id}
+        data-testid={testID}
+        style={logoStyle}
+        x="0px"
+        y="0px"
+        viewBox="0 0 1080 300"
+        width="1080"
```

Comment:
Should size here be controlled by a prop as well? Also, we should lock the aspect ratio so the logo can’t be distorted.

Reply:
These control the internal proportions of the image. If you pass in a style prop with either `width` or `height` it proportionally scales